assets/texts/en/amoDescription.html

```diff
@@ -1 +1,103 @@
+			<li>Convert <a href="https://en.wikipedia.org/wiki/Number_Forms">fractions</a> and <a href="https://en.wikipedia.org/wiki/Mathematical_constant">mathematical constants</a> to Unicode characters (i.e. <code>1234.25</code>
+				with 1234¼, etc.). Supports all Unicode fraction characters, Pi and e constants.</li>
+			<li>Supports undoing autocorrections</li>
+		</ul>
+	</li>
+	<li>Context menu with selected/highlighted text:<ul>
+			<li><a href="https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols">Unicode font</a> conversion (i.e. ꜱᴍᴀʟʟ ᴄᴀᴘꜱ, 𝒮𝒸𝓇𝒾𝓅𝓉, 𝔉𝔯𝔞𝔨𝔱𝔲𝔯, etc.). Supports more than 20 fonts.</li>
+			<li>Change <a href="https://en.wikipedia.org/wiki/Letter_case#Case_styles">casing</a> (i.e. UPPER CASE, lower case, tOGGLE cASE, etc.). Supports all locales.</li>
+			<li>Optionally show a live preview.</li>
+		</ul>
+	</li>
+	<li>Support websites as best as possible by providing website-specific possibles, if needed. (<a href="https://github.com/rugk/unicodify/issues">if there is a problem, please report it</a>)</li>
+	<li>Very fast, even when typing on large documents.</li>
+	<li>Supports the light/dark mode of your system automatically.</li>
+	<li>Settings automatically synced between all browser instances and devices (Firefox and Chrome only).</li>
+	<li>Follows the <a href="https://design.firefox.com/photon">Firefox</a> and <a href="https://style.thunderbird.net/">Thunderbird</a> Photon Design.</li>
+	<li>Puts your privacy first and does not send data anywhere.</li>
+	<li>Compatible with Firefox for Android.</li>
+	<li>Translated into several languages. <a href="https://github.com/rugk/unicodify/blob/main/CONTRIBUTING.md#Translations">Contribute your own language!</a></li>
+	<li>Settings can be managed by your administrator.</li>
+</ul>
+
+
+
+<b>📝 Development 📝</b>
+The add-on is free/libre open-source software and developed on GitHub. <a href="https://github.com/rugk/unicodify">Fork it on GitHub</a> and contribute.
+<a href="https://github.com/rugk/unicodify/contribute">There are some easy issues to start with.</a>
+
+This extension only works with modern Firefox and Thunderbird v78 or higher.
+
+
+<b>🐬 Contribute 🐬</b>
+You can easily get involved in this project and help. Here are some ideas:
+<ul>
+<li>📃 <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#translations">Translate this add-on into multiple languages!</a></li>
+<li>🐛 <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#coding">Fix some easy issues and get started in add-on development</a> (or just try out the development version)</li>
+<li>💡 <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#need-ideas">Or check out some other add-on issues</a> (or translate them).</li>
+</ul>
+Or, in any case, <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#support-us">support us by spreading the word!</a> ❤️
+
+
+
+<b>🗯️ Permissions 🗯️</b>
```

Comment:
May I suggest a different emoji here then the “[Anger Bubble](https://emojipedia.org/right-anger-bubble/)”: 🗨️, 💬, 📋 or ☑️.

Reply:
oh lol, never noticed that, yeah probably… :sweat_smile: 

Also needs to be changed for all add-ons…